runtime/devnet/src/extensions/v0/mod.rs

```diff
@@ -1,2 +1 @@
-pub(crate) mod assets;
 pub(crate) mod error;
```

Comment:
v0/mod.rs is effectively empty, with error.rs as a submodule. It would be far simpler to just have v0.rs with the functionality (a few functions and consts) from error.rs included directly within v0.rs.

Reply:
Refactored